docker-compose.yml

```
@@ -0,0 +1,37 @@
+      - MYSQL_ROOT_PASSWORD=1234
+    volumes:
+      - mysql-data:/var/lib/mysql
+    networks:
+      - sbs-network
+
+networks:
+  sbs-network:
+    driver: bridge
+
+volumes:
+  mysql-data:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)